aiida/manage/manager.py

```diff
@@ -206,7 +209,7 @@ def get_backend(self) -> 'Backend':
 
         """
         if self._backend is None:
-            self._load_backend()
+            return self._load_backend()
```

Comment:
Why is this change necessary?

Reply:
Okay, I guess this is the place where the type assertion properly takes effect. We have:
```python
return cast('Backend', self._backend)
```

In that case, I propose `_load_backend` method should have no return value.